src/coreComponents/physicsSolvers/wavePropagation/AcousticVTIFields.hpp

```diff
@@ -33,29 +33,53 @@ namespace fields
 namespace acousticvtifields
 {
 
-DECLARE_FIELD( Delta,
-               "delta",
+DECLARE_FIELD( AcousticVelocity,
```

Comment:
Same as for the TTI fields, this file should only contains specific VTI variables